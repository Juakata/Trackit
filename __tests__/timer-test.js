import React from 'react';
import renderer from 'react-test-renderer';

import Timer from '../src/components/timer';

it('RoundButton snapShot', () => {
  const snap = renderer.create(
    <Timer interval={0} />,
  ).toJSON();
  expect(snap).toMatchSnapshot();
});
