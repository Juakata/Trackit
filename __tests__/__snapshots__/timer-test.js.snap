// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RoundButton snapShot 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "flexWrap": "wrap",
      "justifyContent": "center",
      "width": "80%",
    }
  }
>
  <Text
    style={
      Object {
        "borderColor": "#006fb0",
        "borderRadius": 10,
        "borderWidth": 2,
        "fontSize": 35,
        "fontWeight": "bold",
        "margin": 3,
        "padding": 10,
        "textAlign": "center",
      }
    }
  >
    00
  </Text>
  <Text
    style={
      Object {
        "fontSize": 30,
        "marginTop": 13,
        "padding": 0,
      }
    }
  >
    :
  </Text>
  <Text
    style={
      Object {
        "borderColor": "#006fb0",
        "borderRadius": 10,
        "borderWidth": 2,
        "fontSize": 35,
        "fontWeight": "bold",
        "margin": 3,
        "padding": 10,
        "textAlign": "center",
      }
    }
  >
    00
  </Text>
  <Text
    style={
      Object {
        "fontSize": 30,
        "marginTop": 13,
        "padding": 0,
      }
    }
  >
    :
  </Text>
  <Text
    style={
      Object {
        "borderColor": "#006fb0",
        "borderRadius": 10,
        "borderWidth": 2,
        "fontSize": 35,
        "fontWeight": "bold",
        "margin": 3,
        "padding": 10,
        "textAlign": "center",
      }
    }
  >
    00
  </Text>
</View>
`;
